core/src/main/resources/org/broadinstitute/dsde/workbench/hamm/liquibase/changesets/20190225_create_tables.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These are the changesets for creating the `Job` and `Workflow` table and any unique or foreign keys. I took these from the doobie table definitions. I added a foreign key on `workflowId` because every job should belong to a workflow.

Questions:
- The naming for the job table columns is confusing. What is `CALL_FQN`, `ATTEMPT`, "VENDOR_JOB_ID" and `JOB_INDEX`? 
- Question from Doug - Is `IS_SUBWORKFLOW` necessary?

Reply:
1. Good point - but in order to tell a user how much a particular job/task cost, we need to be able to check against a workflow collection in sam, which means that the job needs to be associated with a workflow in order for us to authenticate. 

2. No, the operation id cost idea was based on a hunch by someone and it wasn't a real thing :( . I still don't have a clear understanding of what CALL_FQN and ATTEMPT mean - can you explain what they are? 

3.  I can keep `IS_SUBWORKFLOW` for now, we can delete it later if we don't need it

Reply:
`CALL_FQN` is just a name for the job(but it's not unique across workflow, hence you need the combined key), `attempt` is how many times this job has been retried.

Reply:
@ansingh7115  All of those but `VENDOR_JOB_ID` come from Cromwell

Reply:
So in cromwell metadata that starts like this: 
` """
      |{
      |  "actualWorkflowLanguage": "WDL",
      |  "actualWorkflowLanguageVersion": "draft-2",
      |  "calls": {
      |    "echo_strings.echo_files": [
      |      {
      |        "attempt": 1,
      |        "backend": "JES",
      |        "backendLabels": {
      |          "cromwell-workflow-id": "cromwell-0942e6dc-2a2e-4912-86e2-b91fdaf06c44",
      |          "wdl-task-name": "echo-files"
      |        },
      |        "backendLogs": {
      |          "log": "gs://fc-d28aeae4-30cc-46ca-986d-2ffda410f47c/e9e9d8ef-7ca0-4c69-abc2-e93e959c81a8/echo_strings/c5b6ee46-1f09-4830-91a8-fd814866d664/call-echo_files/echo_files.log"
      |        },`

the `CALL_FQN` would be "echo_strings.echo_files"? I'm just trying to pinpoint exactly what it is so I know where it's coming from